install/operator/pkg/syndesis/action/install.go

```diff
@@ -170,22 +151,37 @@ func (a *installAction) Execute(ctx context.Context, syndesis *v1alpha1.Syndesis
 			return err
 		}
 	}
+	addRouteAnnotation(original, syndesisRoute)
+
+	// The above section of code update the syndesis.Spec, lets not modify the CR
+	// lets copy it to the ReifiedSpec field so users can see how it was reified but not change the original
+	// spec on the CR
+	syndesis.ReifiedSpec = syndesis.Spec.DeepCopy()
+	original.ReifiedSpec = syndesis.Spec.DeepCopy()
```

Comment:
Once we have copied it, do we actually do anything with it? If it is purely for user analysis, could it not be dumped to a log file rather than sitting in the CR?

Reply:
User might not have access to operator logs.